### ISSUE TITLE
Features page sometimes comes up blank

### ISSUE THREAD

**Issue by jpmedley** [GoogleChrome/chromium-dashboard#737]
I've seen this behavior with both the /features path generally and on urls with an entry number.

In two weeks I'll be publishing a [beta release post](https://blog.chromium.org/2019/12/chrome-80-content-indexing-es-modules.html). This always links to multiple features on Chrome Status. Since this is designed to attract the attention of the tech press, it would not be good to publish something with links that seem not to work.

In other words, this is somewhat urgent.

![image](https://user-images.githubusercontent.com/8204171/73675773-ec677380-4667-11ea-95b9-53c13ba26724.png)

**Comment by jrobbins** [comment]
Are you able to reproduce the problem?

Any hints about how I might repro it?  Were you signed in or out at the time?  Did it happen like 1 out 4 times or like 1 out of every 10 times?  Which browser were you using?  If you see it happen again, please share any error message from the browser console.

I just tried a bunch of clicking around in Chrome and Safari and I sometimes got pauses but the page always loaded eventually and I don't see any errors on the console.  

The server logs have a bunch of errors related to notifications, but that has been happening for many months.  I don't see anything related to viewing features or anything new.

**Comment by jpmedley** [comment]
It happens intermittently. I couldn't even reproduce its occurrences (yes, it happened more than once) when I repeated my interactions within seconds of the first failure.

I clearly wasn't thinking when I opened this yesterday. I was on a bus to Napa valley. I wonder if I was dealing with an interrupted network connection. So maybe the service worker was showing me the boilerplate and then not indicating that the network connection was flaking?

**Comment by jpmedley** [comment]
It just happened to me again on the Google internal WiFi. It was this link.

https://www.chromestatus.com/features/6068482055602176

When I clicked the link a second time, it loaded. The Network panel says it loaded.

![image](https://user-images.githubusercontent.com/8204171/74044776-7a9d6b80-4980-11ea-9ae5-ad3a3c453abb.png)

There are no errors in the console.

I've attached the contents of the source pane.

[chrome-status.txt](https://github.com/GoogleChrome/chromium-dashboard/files/4171797/chrome-status.txt)

I've left the contents of the errant window open. Is there anything else you want me to look at in DevTools?

**Comment by jrobbins** [comment]
After a bunch more trying, I was able to hit it in chrome 79.  However, the error is on a line that is compressed JS included in an HTML file.  I'll try to deploy a non-compiled version so that I can get a more specific error message.
<img width="1688" alt="Screen Shot 2020-02-07 at 11 44 29 AM" src="https://user-images.githubusercontent.com/17365/74060869-b98ee980-499f-11ea-85d8-12a15b9bf5bd.png">

**Comment by jrobbins** [comment]
I was not able to repro the problem again, but I have deployed a new version of the app that uses some non-compressed JS for that part.   So, next time that I can trigger the error I should be able to debug it.

**Comment by jrobbins** [comment]
I am seeing occasional errors when loading the FCM SDK.  However, with that error, the feature list still renders, so there must be another possible error.

**Comment by jrobbins** [comment]
Today, I worked reproducing this again and didn't have any luck.  I haven't done anything to fix it yet, I am still trying to investigate.

I'm thinking that I could go through all the JS code and add more exception handling at all the places where the app makes web requests.  That way, if one fails, at least we could pop up a notification to tell the user to reload.

One of the main requests that the frontend makes is to FCM for the push notifications.  I'm going to switch back to working on replacing those with email so that it is one less thing that can go wrong.

**Comment by jpmedley** [comment]
I don't think I've seen this in the last month.

**Comment by jrobbins** [comment]
Awesome, thanks!
